src/libs/api/widgetSetting.ts

```diff
@@ -231,7 +231,8 @@ function getParamsByStrategy(
     }
   } else {
     if (useCachedSession) {
-      return { userId: params.userId };
+      return {};
+      // return { userId: params.userId };
```

Comment:
is this because manual reuses channel only?

Reply:
![image](https://github.com/sendbird/chat-ai-widget/assets/26326015/613f7325-6549-44dd-af2f-188e40a33319)